plugins/com.google.cloud.tools.eclipse.appengine.newproject/plugin.properties

```diff
@@ -0,0 +1,5 @@
+google = Google
+Bundle-Vendor = Google Inc.
+Bundle-Name = Wizards for Google Cloud Platform Apps
```

Comment:
"Wizards for Google Cloud Platform's App Engine"?

An aside, but the difference between bundle-name, symbolic-name, and id:
- the Bundle-Symbolic-Name (BSN) is a partial identifier
- each bundle is uniquely identified by its BSN _and_ its version number
   - given two bundles with the same BSN and version, the OSGi framework assumes they are identical and won't load the second
- each bundle when loaded into the OSGi Framework is assigned a unique numeric id
- the Bundle-Name is a human meaningful term and ignored by the framework; the Bundle Name doesn't show up in much of the UI

You'll often see the bundle id in logs like the following: `Could not resolve module: org.eclipse.pde.api.tools.tests [670]`.